SpatialGDK/Documentation/content/command-line-arguments.md

```diff
@@ -0,0 +1,225 @@
+            <td style="width: 62%;">The path that the protocol logs are generated to.
+                <p><strong>Note</strong>: If you enable protocol logging, you must set this value.</p>
+            </td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>-OverrideSpatialNetworking</code></td>
+            <td style="width: 8%;">bool</td>
+            <td style="width: 62%;">Use this argument to override any other setting that you&#39;ve configured for the networking mode:
+                <ul>
+                    <li>When you run the command with the <code>-OverrideSpatialNetworking</code> or the <code>-OverrideSpatialNetworking=true</code> argument, you enable SpatialOS networking.</li>
+                    <li>When you run the command with the <code>-OverrideSpatialNetworking=false</code> argument, you disable SpatialOS networking and use native Unreal networking instead.</li>
+                </ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Map travel URL options
+In the GDK, so that a client-worker instance executes a [map travel]({{urlRoot}}/content/map-travel), you can add the following URL options to the `ClientTravel` URL:
+
+<table style="width: 100%;">
+    <tbody>
+        <tr>
+            <td style="width: 30%;"><strong>Option</strong></td>
+            <td style="width: 8%;"><strong>Type</strong></td>
+            <td style="width: 62%;"><strong>Description</strong></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>locator<code></td>
+            <td style="width: 8%;">flag</td>
+            <td style="width: 62%;">
+                <p dir="ltr">To connect a client-worker instance to a deployment using Locator, add this flag to the <code>ClientTravel</code> URL.</p>Note: If you add this option, ensure that you also add the <code>playeridentity</code> option and the <code>login</code> option to the URL.<br></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>playeridentity</code></td>
+            <td style="width: 8%;">string</td>
+            <td style="width: 62%;">The <a href="https://docs.improbable.io/reference/13.7/shared/auth/integrate-authentication-platform-sdk#2-generating-a-playeridentitytoken"><code>PlayerIdentityToken</code></a> (PIT) that the client-worker instance passes to the Locator service to prove its player identity within SpatialOS.<br></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>login<code><br></td>
+            <td style="width: 8%;">string</td>
+            <td style="width: 62%;">The <a href="https://docs.improbable.io/reference/13.7/shared/auth/integrate-authentication-platform-sdk#3-generating-a-logintoken"><code>LoginToken</code></a> that the client-worker instance passes to the Locator service to be permitted to join the deployment.<br></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>useExternalIpForBridge</code><br></td>
```

Comment:
It looks like this entry is the same as the one in "Connection arguments" - is this intentional?

Reply:
The option belongs specifically to *Map travel URL options*, not to *Connection arguments*. The usage of it is different as well (without `-` in the option Vs with `-` in the argument in the *Connection arguments*).

Reply:
Ah OK, thanks for the explanation! Makes sense :)